Simple tokens added

akroii opened the pull request.

akroii committed:
Simple tokens added

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich jedich nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgegeben werden können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank de ...
```
```suggestion
Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich allerdings nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgeben können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank der Symfony Expression Language können die Einsatzzwecke noch erweitert werden.
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich jedich nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgegeben werden können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank de ...
+
```
```suggestion


```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich jedich nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgegeben werden können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank de ...
+
+## Wie verwende ich Simple Tokens? ##
```
```suggestion
## Wie verwende ich Simple Tokens?
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+
+## Wie verwende ich Simple Tokens? ##
+Simple Tokens beginnen und enden mit zwei Rauten `##`. Welche Werte ausgegeben werden können, steht in der jeweiligen Datenbank-Spalte.
+
```
```suggestion


```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+## Wie verwende ich Simple Tokens? ##
+Simple Tokens beginnen und enden mit zwei Rauten `##`. Welche Werte ausgegeben werden können, steht in der jeweiligen Datenbank-Spalte.
+
+## Welche Simple Tokens stehen zur Verfügung? ##
```
```suggestion
## Welche Simple Tokens stehen zur Verfügung?
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+Simple Tokens beginnen und enden mit zwei Rauten `##`. Welche Werte ausgegeben werden können, steht in der jeweiligen Datenbank-Spalte.
+
+## Welche Simple Tokens stehen zur Verfügung? ##
+Simple Tokens können jedoch nur auf dafür vorgesehene Backend-Module bzw. Extensions verwendet werden. Sofern ein Simple Token nicht unterstützt wird, erfolgt eine leere Ausgabe und ein Log-Eintrag, dass ein Simple Token nicht ersetzt werden konnte.
```
```suggestion
Simple Tokens können nur auf dafür vorgesehene Backend-Module bzw. Extensions verwendet werden. Sofern ein Simple Token nicht unterstützt wird, erfolgt eine leere Ausgabe und ein Log-Eintrag, dass ein Simple Token nicht ersetzt werden konnte.
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+| `##flang##`         | Aktuell verwendete Sprache                                |                             |
+| `##domain##`        | Aktuelle Domain                                           | Newsletter                  |
+| `##link##`          | Link zum Newsletter                                       | Newsletter                  |
+| `##channels##`      | Abonierter News-Channel                                   | Newsletter                  |
```
```suggestion
| `##channels##`      | Abonnierter News-Channel                                   | Newsletter                  |
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+| `##channels##`      | Abonierter News-Channel                                   | Newsletter                  |
+
+Simple Tokens sind seit der Version Contao 2.x fester Bestandteil von Contao. Seit der Version 4.12 sind diese Bestandteil der Symphony Expression Language.
+
```
```suggestion


```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+
+Simple Tokens sind seit der Version Contao 2.x fester Bestandteil von Contao. Seit der Version 4.12 sind diese Bestandteil der Symphony Expression Language.
+
+## Wo können Simple Tokens eingesetzt werden? ##
```
```suggestion
## Wo können Simple Tokens eingesetzt werden?
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+## Wo können Simple Tokens eingesetzt werden? ##
+- Benutzer: Modultyp 'Registrierung'
+- Benutzer: Modultyp 'Passwort vergessen'
+- Newsletter: 'Newsletterleser', 'Abonieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
```
```suggestion
- Newsletter: 'Newsletterleser', 'Abonnieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+- Benutzer: Modultyp 'Registrierung'
+- Benutzer: Modultyp 'Passwort vergessen'
+- Newsletter: 'Newsletterleser', 'Abonieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
+- Extensions z.B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads
```
```suggestion
- Extensions z. B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+- Newsletter: 'Newsletterleser', 'Abonieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
+- Extensions z.B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
```
```suggestion


```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+- Extensions z.B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
+## Weitere Einsatzzwecke: ##
```
```suggestion
## Weitere Einsatzzwecke:
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
+## Weitere Einsatzzwecke: ##
+`datei_von_##tstamp##.pdf` für Datein erzeugt datei_von_1650437899.pdf
```
```suggestion
`datei_von_##tstamp##.pdf` für Dateien erzeugt datei_von_1650437899.pdf
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+## Weitere Einsatzzwecke: ##
+`datei_von_##tstamp##.pdf` für Datein erzeugt datei_von_1650437899.pdf
+
+`files/data/##form_broschuere##.pdf` Pfad zur PDF
```
```suggestion
`files/data/##form_broschuere##.pdf` Pfad zur PDF-Datei
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+
+`files/data/##form_broschuere##.pdf` Pfad zur PDF
+
+Es ist ebenso möglich, Simple Tokens als Fallabfrage zu verwenden. Hierzu werden Simple Tokens nicht mehr mit `##` geschrieben, sondern mit beginnender und endender Klammer `{}`
```
```suggestion
Es ist ebenso möglich, Simple Tokens als Fallabfrage zu verwenden. Hierzu werden Simple Tokens nicht mehr mit `##` geschrieben, sondern mit öffnender und schließender Klammer `{}`
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
````diff
@@ -0,0 +1,90 @@
+Beispiel für das Newsletter-Modul:
+```
+{if flang==en}
+  Your language is english
````
```suggestion
  Your language is English.
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+{if flang==en}
+  Your language is english
+{elseif==de}
+  Deine Sprache ist deutsch
```
```suggestion
  Deine Sprache ist Deutsch.
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+{elseif==de}
+  Deine Sprache ist deutsch
+{else}
+  Coldn't assign a language
```
```suggestion
  Couldn't assign a language.
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
````diff
@@ -0,0 +1,90 @@
+  Coldn't assign a language
+{endif}
+```
+
````
```suggestion


```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
````diff
@@ -0,0 +1,90 @@
+{endif}
+```
+
+## Welche Operatoren können bei Beindungen gesetzt werden? ##
````
```suggestion
## Welche Operatoren können bei Bedingungen gesetzt werden?
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+| `<=`        | Kleiner gleich                |
+| `>=`        | Größer gleich                 |
+
+Generell empfiehlt es sich keine Leerzeichen vor den Operatoren zu setzen. Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```
```suggestion
Generell empfiehlt es sich, keine Leerzeichen vor den Operatoren zu setzen. Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```

xchs commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+
+`{if value=="foo" || value=="bar"}`
+
+Es kann auch überprüft werden ob ein gewisser Wert gesetzt ist:
```
```suggestion
Es kann auch überprüft werden, ob ein gewisser Wert gesetzt ist:
```

akroii commented:
Thank you @xchs

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,90 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich jedich nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgegeben werden können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank de ...
```
Das Hauptproblem mit diesem Artikel ist für mich, dass Simple Tokens ein generelles Konzept sind. Sie werden hier allerdings so beschrieben, als würden sie sich direkt auf irgendwelche Datenbank-Felder beziehen. Das ist aber keineswegs der Fall. Simple Tokens sollten m.M.n. als generelles Kapitel erklärt werden. Welche Tokens dabei zur Verfügung stehen, entscheidet der/die Entwickler/in. Ob es Tokens sind, die Datenbankfelder repräsentieren oder Formulardaten oder was auch immer, ist völlig egal für die Erklärung des generellen Konzepts :-)

akroii replied:
Dieser Einleitungstext wurde mittlerweile schon von jemanden bearbeitet.

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update introduction

Simple tokens do not only represent database fields, but must be defined by the developer.

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Hannes <1218766+xchs@users.noreply.github.com>

akroii committed:
Update simple-tokens.de.md

akroii committed:
Paragraphs unified

akroii reviewed (commented):
Typos fixed, paragraphs unified

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,95 @@
+| `<=`        | Kleiner gleich                |
+| `>=`        | Größer gleich                 |
+
+Generell empfiehlt es sich, keine Leerzeichen vor den Operatoren zu setzen. Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```
Ich nutze immer Leerzeichen, warum empfiehlt es sich nicht, die zu nutzen? Ist doch viel lesbarer :D Ich weiss nicht mehr genau aber kann sein, dass es in <4.12 nicht ging. Aber für alle die auf der aktuellen 4.13 sind, würde ich schon empfehlen, Leerzeichen zu nutzen.

fiedsch replied:
In Ergänzung zu Yanicks Anmerkungen: wenn es (in <4.12) nicht ging, wäre es m.E. korrekt zu sagen: "dürfen keine Leerzeichen verwendet werden". Die Formulierung "empfiehlt" impliziert für mich, daß beides geht, aber eines -- warum auch immer -- besser ist.

akroii replied:
Ich habe es jetzt einmal in der 4.13 ausprobiert. Es funktioniert beides, sowohl mit Leerzeichen als auch ohne.

fritzmg replied:
@akroii funktioniert es auch in 4.9?

akroii replied:
@fritzmg ja, funktioniert auch in 4.9. Ist also egal, ob nun mit oder ohne Leerzeichen

fritzmg replied:
Dann muss dieser Satz entfernt werden:

> Generell empfiehlt es sich, keine Leerzeichen vor den Operatoren zu setzen.

akroii replied:
erledigt

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
````diff
@@ -0,0 +1,95 @@
+Beispiel für das Newsletter-Modul:
+
+```
+{if flang==en}
````
```suggestion
{if flang=="en"}
```

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
````diff
@@ -0,0 +1,95 @@
+```
+{if flang==en}
+  Your language is English.
+{elseif==de}
````
```suggestion
{elseif=="de"}
```

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Yanick Witschi <yanick.witschi@terminal42.ch>

akroii committed:
Update docs/manual/article-management/simple-tokens.de.md

Co-authored-by: Yanick Witschi <yanick.witschi@terminal42.ch>

akroii committed:
simple tokens are possible in MP_Forms too

akroii committed:
Using operators with or without empty spaces

akroii committed:
typos fixed, operator in syntax added

akroii reviewed (commented):
Improve readability, operator added

akroii commented:
Gibt es noch Anmerkungen? Bin jetzt erstmal fertig. Alles Weitere (z.b. Symphony expression language und regex) wird dann Schritt für Schritt ergänzt.

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, Inhalte ausgeben können. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte ausgeben können, sondern auch Werte mit Fallabfragen und regulären Ausdrücken analysiert werden können. Simple Tokens sind seit der Version Contao 2.x fester Bestandteil von Contao. Die Entwickler entscheiden, welche Simple Tokens zur Verfügung steh ...
```
```suggestion
Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, Inhalte ausgeben können. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte ausgeben können, sondern auch Werte mit Fallabfragen und regulären Ausdrücken analysiert werden können. Simple Tokens sind seit der Version Contao 2.x fester Bestandteil von Contao. Im Gegensatz zu Insert-Tags, die global eingesetzt werden können, entscheiden bei Simple Tokens die Entwickler*innen, welche Simple Tokens zur Verfügung stehen und wo diese eingesetzt werden können. Es empfiehlt sich in die jeweilige Dokumentation der Extension hineinzusehen. Ab Contao 4.10 nutzen Simple Tokens die Symfony Expression Language, mit der die Einsatzzwecke noch erweitert werden können.
```

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+
+
+## Wie verwende ich Simple Tokens?
+Simple Tokens beginnen und enden mit zwei Rauten `##`. Welche Werte ausgegeben werden können, ist in dem jeweiligen Backend-Modul oder Extension zu finden.
```
```suggestion
Die Ausgabe von Simple Tokens erfolgt mit zwei umschließenden Rauten `##`. Welche Werte ausgegeben werden können, ist in dem jeweiligen Backend-Modul oder Extension zu finden. Innerhalb von Fallabfragen, werden die Rauten weggelassen.
```

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+| `##flang##`         | Aktuell verwendete Sprache                                |                             |
+| `##domain##`        | Aktuelle Domain                                           | Newsletter                  |
+| `##link##`          | Link zum Newsletter                                       | Newsletter                  |
+| `##channels##`      | Abonnierter News-Channel                                   | Newsletter                  |
```
```suggestion
| `##channels##`      | Abonnierter News-Channel                                  | Newsletter                  |
```

Toflar commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+
+Die Operatoren können mit oder ohne Leerzeichen gesetzt werden. Es funktioniert daher sowohl `{if form_name != ""}` als auch `{if form_name!=""}`.
+
+Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```
That only works as of Contao 4.10 and Symfony Expression Language.

akroii replied:
`{if form_name != ""}` and `{if form_name!=""}` also work in 4.9

The operators `||` and `&&` actually do not work in 4.9 yet

Toflar replied:
Yes, that's what I'm saying.

akroii replied:
done

akroii reviewed (commented):
Improve readabillity, fix misstatement

akroii committed:
Info about Version added

akroii reviewed (commented):
Version-info for operators added

fritzmg commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, Inhalte ausgeben können. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte ausgeben können, sondern auch Werte mit Fallabfragen und regulären Ausdrücken analysiert werden können. Simple Tokens sind seit der Version Contao 2.x fester Bestandteil von Contao. Die Entwickler entscheiden, welche Simple Tokens zur Verfügung steh ...
+
+
+## Wie verwende ich Simple Tokens?
```
```suggestion
## Wie verwende ich Simple Tokens?

```

fritzmg commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+Simple Tokens beginnen und enden mit zwei Rauten `##`. Welche Werte ausgegeben werden können, ist in dem jeweiligen Backend-Modul oder Extension zu finden.
+
+
+## Welche Simple Tokens stehen zur Verfügung?
```
```suggestion
## Welche Simple Tokens stehen zur Verfügung?

```

fritzmg commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+| `##channels##`      | Abonnierter News-Channel                                   | Newsletter                  |
+
+
+## Wo können Simple Tokens eingesetzt werden?
```
```suggestion
## Wo können Simple Tokens eingesetzt werden?

```

fritzmg commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,98 @@
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
+
+## Weitere Einsatzzwecke:
```
```suggestion
## Weitere Einsatzzwecke:

```

fritzmg reviewed (commented):
Some formatting nit picks.

akroii committed:
Add newlines for headlines

akroii reviewed (commented):
Add newlines for headlines

akroii commented:
Can it be merged?
I will add more infos about regex for simple tokens in future.

akroii reviewed (commented):
Improve introduction. Made some sentences more clear.

PS: sorry, made a mistake,... I've pressed the wrong button 🙈

fritzmg commented:
> Can it be merged?

You still need to address the requested changes.

akroii committed:
Apply suggestions from code review

Improve introduction. Made some sentences more clear.

Co-authored-by: Yanick Witschi <yanick.witschi@terminal42.ch>

fritzmg commented on docs/manual/article-management/simple-tokens.de.md:
```diff
@@ -0,0 +1,102 @@
+
+Die Operatoren können mit oder ohne Leerzeichen gesetzt werden. Es funktioniert daher sowohl `{if form_name != ""}` als auch `{if form_name!=""}`.
+
+Ab 4.13 kann mittels `&&` und `||` auch mehrere Werte per AND/OR abgefragt werden:
```
```suggestion
Ab 4.13 können mittels `&&` und `||` auch mehrere Werte per AND/OR abgefragt werden:
```

akroii committed:
change wording

Co-authored-by: Fritz Michael Gschwantner <fmg@inspiredminds.at>

netzarbeiter committed:
Update simple-tokens.de.md

netzarbeiter reviewed (approved):
Thank you, Andreas

netzarbeiter commented:
https://github.com/contao/docs/commit/98bc6bc3605db0b3fefd3e95e104b2c7f36dbd85
